redis_store_test.go

```diff
@@ -233,6 +245,9 @@ func TestUserPresence(t *testing.T) {
 
 	store.MarkInactive("bazbar", "sock1")
 
+	// Give it time to propagate to Redis...
+	time.Sleep(1 * time.Second)
```

Comment:
yeah, I'll refactor it so all the async commands return a channel that I can pull from to block.